continuous_integration/environment-3.7.yaml

```diff
@@ -6,7 +6,7 @@ dependencies:
   # required dependencies
   - python=3.7.*
   - numpy=1.17.*
-  - pandas=0.25.*
+  - pandas=1.0.*
```

Comment:
Maybe this should be 0.25.* and the environment-3.8 should be 1.2.*

Reply:
Yeah, I was planning to switch env-3.8 to 1.2.*, but I first want to get everything green with 0.25.*/1.0.*/1.1.*, before dropping one of those to replace with 1.2.*

Reply:
:100: that sounds like a great plan :)